Mathlib/Algebra/Parity.lean

```diff
@@ -510,48 +526,79 @@ end Ring
 
 section Powers
 
-set_option linter.deprecated false
+section LinearOrderedSemiring_ExistsAddOfLE
```

Comment:
@YaelDillies, can you add a comment to this section explaining why this is more interesting than `CanonicallyOrderedCommSemiring`?

Reply:
You mean `CanonicallyLinearOrderedCommSemiring` (which doesn't exist)?